jtl/httphandlers.go

```diff
@@ -36,9 +36,13 @@ func StatusHandler(w http.ResponseWriter, r *http.Request) {
 	state["Version"] = GetConfig(ctx).Version
 	state["Config"] = GetConfig(ctx)
 	callstats := make(map[string]interface{}, 0)
-	if ctx.Value(EelDispatcher) != nil {
-		callstats["WorkQueueFillLevel"] = len(GetWorkDispatcher(ctx).WorkQueue)
-		callstats["WorkersIdle"] = len(GetWorkDispatcher(ctx).WorkerQueue)
+	tenantId := ""
+	if ctx.Value(EelTenantId) != nil {
+		tenantId = Gctx.Value(EelTenantId).(string)
```

Comment:
Does that mean when we query status, we need to pass in tenant? Doesn't it make sense to show status to all tenants?